replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorRunner.java

```diff
@@ -108,13 +121,32 @@ void scheduleReplication() {
             .objects()
             .filter(c -> !c.isSuspended())
             .collect(Collectors.toList());
+
     try {
       for (ReplicatorConfig config : configsToSchedule) {
+        if (sourceOrDestinationIsRemotelyManaged(config)) {
+          LOGGER.trace(
+              "One of config {}'s sites are remotely managed, not running the config",
+              config.getName());
+          continue;
+        }
         replicator.submitSyncRequest(
             new SyncRequestImpl(config, new ReplicationStatusImpl(config.getName())));
       }
     } catch (InterruptedException e) {
       Thread.currentThread().interrupt();
     }
   }
+
+  private boolean sourceOrDestinationIsRemotelyManaged(ReplicatorConfig replicatorConfig) {
+    try {
+      return siteManager.get(replicatorConfig.getSource()).isRemoteManaged()
+          || siteManager.get(replicatorConfig.getDestination()).isRemoteManaged();
+    } catch (NotFoundException | ReplicationPersistenceException e) {
+      LOGGER.debug(
+          "Unable to determine if replication {} should be run based on its sites. This replication will not be run.",
+          replicatorConfig.getName());
+      return true;
```

Comment:
Return true here? Should we just catch the error in the calling method and log it?

Reply:
Depends, do we still want it to run locally if we are not able to determine by the sites?

Reply:
I am not sure it's one or the other. When we finally make the decision to send a heartbeat it would be neither of these 2 since I don't think we want to start replication since we were unable to load the site config anyway and we are unable to send the heartbeat because ... we couldn't load the site anyway.
Does it really make sense to have this method to start with knowing what is coming? I think we might just want to get a hold of the 2 sites in the above method and make our decision there based on one or the other because down the road (next PR), we will need to save the site we need to send a heartbeat in a set such that after the loop we can send the heartbeats (or request them to be sent).